bottlecap/src/config/mod.rs

```diff
@@ -141,6 +139,7 @@ impl Default for Config {
             logs_config_processing_rules: None,
             logs_config_use_compression: true,
             logs_config_compression_level: 6,
+            logs_config_logs_dd_url: String::default(),
```

Comment:
Why making them empty instead of `Option<String>`?

Reply:
I see, so you're re-using it at the end because technically, it will always exist, but we want to ensure that we can set a different one if it exist.

Reply:
Exactly.